src/il_representations/scripts/chain_configs.py

```diff
@@ -0,0 +1,507 @@
+from il_representations.scripts.utils import StagesToRun
```

Comment:
This file seems to be a duplicate of [`configs/chain_configs.py`](https://github.com/HumanCompatibleAI/eirli/blob/cyn-expt/src/il_representations/configs/chain_configs.py). Was it accidentally included in this PR?